src/org/openbravo/erpCommon/ad_process/ConvertQuotationIntoOrder.java

```diff
@@ -113,6 +115,11 @@ public Order convertQuotationIntoSalesOrder(final boolean recalculatePrices,
       final String quotationId) {
     // Create Sales Order
     Order quotation = OBDal.getInstance().get(Order.class, quotationId);
+
+    if (!checkBusinessPartnerInvoiceAddress(quotation)) {
+      throw new OBException(OBMessageUtils.messageBD("NoInvoiceAddressForOrders"));
+    }
+
```

Comment:
If the business partner has an invoice address this address was assigned in the new order?